javascript/load-gen.ts

```diff
@@ -0,0 +1,164 @@
+import {
+  AlreadyExistsError,
+  CacheGetStatus,
+  GetResponse,
+  LogFormat,
+  LogLevel,
+  getLogger,
+  SimpleCacheClient,
+  LoggerOptions,
+  Logger,
+  InternalServerError,
+  TimeoutError,
+} from '@gomomento/sdk';
+
+const authToken = process.env.MOMENTO_AUTH_TOKEN;
+if (!authToken) {
+  throw new Error('Missing required environment variable MOMENTO_AUTH_TOKEN');
+}
+
+const defaultTtl = 60;
+
+const loggerOptions: LoggerOptions = {
+  level: LogLevel.DEBUG,
+  // level: LogLevel.INFO,
+  format: LogFormat.CONSOLE,
+};
+
+const momento = new SimpleCacheClient(authToken, defaultTtl, {
+  requestTimeoutMs: 5 * 1000,
+  loggerOptions: loggerOptions,
+});
+const logger: Logger = getLogger('load-gen', loggerOptions);
+const cacheName = 'js-loadgen';
+
+const main = async () => {
+  try {
+    await momento.createCache(cacheName);
+  } catch (e) {
+    if (e instanceof AlreadyExistsError) {
+      logger.info('cache already exists');
+    } else {
+      throw e;
+    }
+  }
+
+  const numAsyncWorkers = 1000;
+  const numRequestsPerWorker = 50;
+  // const numAsyncWorkers = 1;
+  // const numRequestsPerWorker = 1000;
```

Comment:
Suggestion: delete comments and start with modest numbers, 10 workers and 1000 requests per worker?